{
  "success": false,
  "error": "Traceback (most recent call last):\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\slither\\__main__.py\", line 834, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\slither\\__main__.py\", line 87, in process_all\n    compilations = compile_all(target, **vars(args))\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\crytic_compile\\crytic_compile.py\", line 718, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\crytic_compile\\crytic_compile.py\", line 207, in __init__\n    self._compile(**kwargs)\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\crytic_compile\\crytic_compile.py\", line 629, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\crytic_compile\\platform\\solc.py\", line 151, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\crytic_compile\\platform\\solc.py\", line 280, in _get_targets_json\n    return _run_solc(\n           ^^^^^^^^^^\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\crytic_compile\\platform\\solc.py\", line 497, in _run_solc\n    compiler=\"solc\", version=get_version(solc, env), optimized=is_optimized(solc_arguments)\n                             ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\crytic_compile\\platform\\solc.py\", line 395, in get_version\n    raise InvalidCompilation(\ncrytic_compile.platform.exceptions.InvalidCompilation: \nSolidity version not found:\nSTDOUT:\n\nSTDERR:\nTraceback (most recent call last):\r\n  File \"<frozen runpy>\", line 198, in _run_module_as_main\r\n  File \"<frozen runpy>\", line 88, in _run_code\r\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Scripts\\solc.EXE\\__main__.py\", line 7, in <module>\r\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\solc_select\\__main__.py\", line 94, in solc\r\n    subprocess.run(\r\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\subprocess.py\", line 548, in run\r\n    with Popen(*popenargs, **kwargs) as process:\r\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\subprocess.py\", line 1026, in __init__\r\n    self._execute_child(args, executable, preexec_fn, close_fds,\r\n  File \"C:\\Users\\20520\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\subprocess.py\", line 1538, in _execute_child\r\n    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,\r\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\nOSError: [WinError 193] %1 is not a valid Win32 application\r\n\n",
  "results": {}
}